src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../types/User';
+
+type Props = {
+  user: User | null
```

Comment:
you check if user exist at TodoInfo
```suggestion
  user: User
```